code/modules/events/disease_outbreak.dm

```diff
@@ -22,35 +21,27 @@ GLOBAL_LIST_EMPTY(current_pending_diseases)
 			if(EVENT_LEVEL_MUNDANE)
 				virus = pick(diseases_minor)
 			if(EVENT_LEVEL_MODERATE)
```

Comment:
Should this be switched to `if(EVENT_LEVEL_MODERATE, EVENT_LEVEL_MAJOR)`? Right now it looks like if major is rolled, it makes a stack trace, then makes the virus a cold, which seems odd.

Reply:
Ah, right, I see.